reproman/interface/tests/test_execute.py

```diff
@@ -124,6 +124,8 @@ def test_trace_docker(docker_container, trace_info):
 @pytest.mark.integration
 @mark.skipif_no_network
 @mark.skipif_no_apt_cache
+# Error: trace.sqlite3 doesnt exist in the directory https://github.com/VIDA-NYU/reprozip/issues/387
+@pytest.mark.xfail(reason="Reprozip binary is currently broken.")
```

Comment:
I would just make it a bit more specific and not blame reprozip itself
```suggestion
@pytest.mark.xfail(reason="Our extracted standalone reprozip tracer is currently broken.")
```